src/agent/index.js

```diff
@@ -11,32 +11,26 @@ const {
 } = require('../data')
 
 /**
- * Class representing Gaijin.
+ * Class representing a bot Agent.
  */
-class Gaijin {
+class Agent {
   /**
-   * Create a Gaijin
-   * @param {String} token                                    The token to log in to the Discord API with.
-   * @param {Object} databaseInfo                             The info for the PostgreSQL database.
-   * @param {String} databaseInfo.connectionURL               The PostgreSQL database url.
-   * @param {String} databaseInfo.tableName                   The name of the table we store user data in the PostgreSQL database.
-   * @param {Object} [options={}]                             Options object.
-   * @param {Number} [options.connectRetryLimit=5]            The maximum number of times to retry connecting to the Discord API.
-   * @param {String} [options.prefix='!!!']                   The command prefix.
-   * @param {String} [options.dblToken]                       The token used with the DiscordBotsList API.
-   * @param {Number} [options.remindersCheckInterval=3000000] The amoount of time to wait between checking on reminders.
+   * Create an Agent.
+   * @param {String}          token           The token to log in to the Discord API with.
+   * @param {DatabaseOptions} databaseOptions The info for the database.
+   * @param {AgentOptions}    [agentOptions]  Options for the agent.
    */
-  constructor (token, databaseInfo, options = {}) {
+  constructor (token, databaseOptions, agentOptions = {}) {
     const {
       connectionURL,
-      tableName
-    } = databaseInfo
+      tables
+    } = databaseOptions
     const {
       connectRetryLimit = 5,
```

Comment:
This wasn't increased.

Reply:
oh i updated the docs and forgot the code